### PR TITLE
feat: added new event handle supplementary evidence (FPLA-1079)

### PR DESCRIPTION
**Pull request by swalker125**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/FPLA-1079

### Change description ###

New event handle supplementary evidence.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-definition/AuthorisationCaseField/court-admin.json [hunk 1]
```diff
@@ -635,5 +635,12 @@
     "CaseFieldID": "placementsWithoutPlacementOrder",
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "CUD"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "supplementaryEvidenceHandled",
```
Comment (tomaszpowroznik) [inline ccd-definition/AuthorisationCaseField/court-admin.json]: it must be evidenceHandled

File: ccd-definition/CaseEvent/Submitted.json [hunk 2]
```diff
@@ -272,5 +272,19 @@
...
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "handleSupplementaryEvidence",
+    "Name": "Handle supplementary evidence",
+    "Description": "Handle supplementary evidence",
+    "DisplayOrder": 18,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "*",
```
Comment (tomaszpowroznik) [inline ccd-definition/CaseEvent/Submitted.json]: i would use dedicated file for this, please coordinate what filename it is with Piotr, as he is using something similar
see
https://github.com/hmcts/fpl-ccd-configuration/pull/826/files#diff-9ad485151202c361c067deb09d40a9d9

File: ccd-definition/CaseEventToFields/handleSupplementaryEvidence.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    "CaseEventID": "handleSupplementaryEvidence",
+    "CaseFieldID": "supplementaryEvidenceHandled",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY",
+    "PageID": 1,
+    "PageLabel": "Bulk Scanning ",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1
+  }
+]
```
Comment (tomaszpowroznik) [inline ccd-definition/CaseEventToFields/handleSupplementaryEvidence.json]: new line can be added

File: ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json [hunk 4]
```diff
@@ -82,5 +82,40 @@
...
+    "CaseEventID": "handleSupplementaryEvidence",
+    "UserRole": "caseworker-publiclaw-judiciary",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "handleSupplementaryEvidence",
+    "UserRole": "caseworker-publiclaw-gatekeeper",
+    "CRUD": "R"
```
[the remark this answers is not in the document]
Comment (swalker125) [reply ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json]: I'm going by A/C 6 on the ticket: 

"On the History tab display the event that has happened (Handle supplementary evidence) and all roles should be able to view this."

File: ccd-definition/CaseEvent/AnyState.json [hunk 5]
```diff
@@ -0,0 +1,13 @@
+{
+  "LiveFrom": "01/01/2017",
+  "CaseTypeID": "CARE_SUPERVISION_EPO",
+  "ID": "handleSupplementaryEvidence",
+  "Name": "Handle supplementary evidence",
+  "Description": "Handle supplementary evidence",
+  "PreConditionState(s)": "*",
```
Comment (lewisbirks) [inline ccd-definition/CaseEvent/AnyState.json]: Note that this means that it will be visible in the deleted state

Comment (swalker125) [reply ccd-definition/CaseEvent/AnyState.json]: only solicitor has access to deleted state

**Pull request merged**